pkg/cmd/system.go

```diff
@@ -235,7 +240,11 @@ func NewHelmInstall(version, releaseName string, ioStreams cmdutil.IOStreams) (*
 	client := action.NewInstall(actionConfig)
 	client.Namespace = types.DefaultOAMNS
 	client.ReleaseName = releaseName
-	client.Version = version
+	if len(version) > 0 {
+		client.Version = version
+	} else {
+		client.Version = types.DefaultOAMVersion
+	}
```

Comment:
FYI, reverted the client.version code lines to fix OAM Runtime Chart installation issue.